source/extensions/filters/network/mongo_proxy/codec.h

```diff
@@ -187,6 +187,27 @@ class CommandReplyMessage : public virtual Message {
 
 typedef std::unique_ptr<CommandReplyMessage> CommandReplyMessagePtr;
 
+class MsgMessage : public virtual Message {
+public:
+  struct FlagBits {
+    // clang-format off
+    static const int32_t ChecksumPresent = 0x1 << 0;
+    static const int32_t MoreToCome      = 0x1 << 1;
+    static const int32_t ExhaustAllowed  = 0x1 << 16;
```

Comment:
I think the current practice in Envoy is to use constexpr

Reply:
I did it this way, so that it was more consistent with the other classes in the file that had a Flags struct (i.e. QueryMessage, ReplyMessage).  Either way, it's an easy change.  Should I change it?